Web/HTML1/Assignment1/Index.html

```diff
@@ -0,0 +1,234 @@
+
+<!DocTyoe html>
+
+<html>
+
+<head>
+
+<title>Account Creation</title>
+</head>
+
+<body>
+	<table>
+		<tr>
+			<td colspan="2"><b>Create Your Account:</b></td>
+		</tr>
+
+		<tr>
+			<td align="right">Email ID:</td>
+			<td><input type="email" placeholder="Enter your email"></td>
+		</tr>
+
+		<tr>
+			<td align="right">Password</td>
+			<td><input type="password"></td>
+		</tr>
+
+		<tr>
+			<td align="right">Confirm Password</td>
+			<td><input type="password"></td>
+		</tr>
+
+		<tr>
+			<td colspan="2"><hr></td>
+		</tr>
+
+		<tr>
+			<td colspan=2><b>Contact Details</b></td>
+		</tr>
+
+		<tr>
+			<td align="right">Your Name:</td>
+			<td><input type="text"></td>
+		</tr>
+
+		<tr>
+			<td align="right">Current Location</td>
+			<td><select>
+					<option selected disabled>select</option>
+					<option>jaipur</option>
```

Comment:
Value attribute is missing in option tag. This value is required when we need to store the value in DB. If you not use then it store null.